NEWS.md

```diff
@@ -32,7 +32,7 @@ New library functions
 * The new `only(x)` function returns the one-and-only element of a collection `x`, and throws an `ArgumentError` if `x` contains zero or multiple elements ([#33129]).
 * `takewhile` and `dropwhile` have been added to the Iterators submodule ([#33437]).
 * `accumulate` has been added to the Iterators submodule ([#34033]).
-* There is a now an `evalpoly` function meant to take the role of the `@evalpoly` macro. The function is just as efficient as the macro while giving added flexibility, so it should be preferred over `@evalpoly`. `evalpoly` takes a list of coefficients as a tuple, so where one might write `@evalpoly(x, p1, p2, p3)` one would instead write `evalpoly(x, (p1, p2, p3))` (#[32753]).
```

Comment:
The actual link needs to be added to the bottom of the file, right?

Reply:
According to https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#news-worthy-changes, it does not need to be done in the PR, but I ran the script anyway and made a commit anyway as I am assuming we are nearing an RC.

Reply:
This is making the PR towards the release branch which is why I thought it makes sense to do it here.

Reply:
I totally agree, so now it is done.

Sorry for the mistakes and the noise. I hope it can be merged now without me ending up in the *Most Bugs in a PR per Characters Changed Hall of Fame*.

Reply:
Don't worry, you'll never top https://github.com/JuliaLang/julia/commit/04e1c98a46b5a6d7be62ed472c8a014042b41eb1.